src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -157,9 +157,10 @@
     }
 }
 
-.is-responsive-web {
-    @include breakpoint($medium-screen) {
-        .be .bcs-activity-feed,
+@include breakpoint($medium-screen) {
+    .be .bcs-activity-feed {
+        width: 100%;
+
         .bcs-activity-feed-comment-input {
```

Comment:
I moved `bcs-activity-feed-comment-input` under the `.be .bcs-activity-feed` scope to match the nesting and specificity of the base styles